R/system.R

```diff
@@ -1089,17 +1089,41 @@ select_columns <- function(x, ...) {
   return (df)
 }
 
+clear_cach_file <- function(url){
```

Comment:
Typo "cach".
Also, some comment about how this is used (to cache downloaded remote file etc.),
how it works (uses tempfile https://stat.ethz.ch/R-manual/R-devel/library/base/html/tempfile.html , and a R variable with name of hashed url is assigned to the path given by tempfile.)